### PR TITLE
Issues/issue 92

### PR DESCRIPTION
**Pull request by RomanVolkov**
PR for https://github.com/SwiftWeekly/swiftweekly.github.io/issues/325

### CODE REVIEW

File: _drafts/2017-10-26-issue-92.md [hunk 1]
```diff
@@ -9,50 +9,92 @@ sponsor:
...
 
-> TODO: Latest episode of Swift Unwrapped
+[Episode 33: HTTP server v0.1.0](https://spec.fm/podcasts/swift-unwrapped/89166): Jesse and JP discuss the first release of the HTTP server library from the Swift Server APIs project.
+
+[Episode 34: Floating-Point](https://spec.fm/podcasts/swift-unwrapped/91615): JP and Jesse share their thoughts on numeric protocols and types, especially those of the floating point variety.
 
 ### News and community
 
-> TODO
+Swift Core team [posted](https://swift.org/blog/swift-4-1-release-process/) the article that the goals, release process, and estimated schedule for Swift 4.1. Swift 4.1 is a source compatible update to Swift 4.0, but is not binary compatible with 4.0. Swift 4.1 is intended to be released in the first half of 2018.
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: "the article that the goals"
->
"an article discussing the goals" ?

File: _drafts/2017-10-26-issue-92.md [hunk 2]
```diff
@@ -9,50 +9,92 @@ sponsor:
...
+
+Andre Videla [posted](https://medium.com/@andre_videla/total-programming-in-swift-526508c12a74) discussion of how Swift encourages it's totality. And there's a [discussion](https://twitter.com/clattner_llvm/status/918885734890663937) on Twitter.
+
+[Slava Pestov](https://twitter.com/slava_pestov) [started](https://twitter.com/slava_pestov/status/922357520819134465) interesting accompanied discussion to improvements of type checker performance in non-WMO builds of projects with many classes.
+
+Harlan Haskins made a great [talk](academy.realm.io/posts/improving-swift-tools-with-libsyntax-try-swift-haskin-2017/) about structure of libSyntax, design decisions involved with it, and how to make use of it to analyze, generate, and transform Swift code.
+
+It's never too much to repeat the basics, especially in Swift. Ennio Masi [wrote](https://medium.com/@EnnioMa/back-to-the-fundamentals-sorting-algorithms-in-swift-from-scratch-fccf8a3daea3) all popular sorting algorithms in pure Swift🤓.
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: "all popular" -> "about popular"

**Comment by SD10** [comment]
@RomanVolkov I can keep reviewing this and continue to make suggestions regarding wording if you'd like. I just don't want you to think I'm an 💩 😅 Appreciate your hard work on this 👍

**Comment by BasThomas** [comment]
Thanks for checking, @SD10! 🎉

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: _drafts/2017-10-26-issue-92.md [hunk 3]
```diff
@@ -9,50 +9,92 @@ sponsor:
...
+
+[Slava Pestov](https://twitter.com/slava_pestov) [started](https://twitter.com/slava_pestov/status/922357520819134465) interesting accompanied discussion to improvements of type checker performance in non-WMO builds of projects with many classes.
+
+Harlan Haskins made a great [talk](academy.realm.io/posts/improving-swift-tools-with-libsyntax-try-swift-haskin-2017/) about structure of libSyntax, design decisions involved with it, and how to make use of it to analyze, generate, and transform Swift code.
+
+It's never too much to repeat the basics, especially in Swift. Ennio Masi [wrote](https://medium.com/@EnnioMa/back-to-the-fundamentals-sorting-algorithms-in-swift-from-scratch-fccf8a3daea3) about popular sorting algorithms in pure Swift🤓.
+
+If you still doubted to use Swift - check [this](https://medium.com/@guydaher/what-stats-and-surveys-are-saying-about-swift-in-2017-7e21dcce1f8b) stats and surveys around Swift. Swift's position is stronger and stronger💪.
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: "If you still doubted to use Swift - check [this]"
->
"If you're still doubting Swift - check [these]"

File: _drafts/2017-10-26-issue-92.md [hunk 4]
```diff
@@ -9,50 +9,92 @@ sponsor:
...
+
+Xiaodi Wu [continued](https://lists.swift.org/pipermail/swift-dev/Week-of-Mon-20171016/005629.html) the conversation about rationalizing FloatingPoint conformance to Equatable at [swift-dev].
+> Regardless of whether or not
+> it is wise for the partial equivalence relation `FloatingPoint.==` to be
+> deemed as semantically satisfactory for the purposes of `Equatable`, the
+> fact remains that Swift deliberately ships with such a conformance, and
+> `Array.==` should not blow up given that explicitly contemplated design
+> decision. This PR resolves that proximal issue.
+
+Max Moiseev [started a discussion](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20171023/040609.html) with introduction of new standart sequence method: Sequence.filteredMap(_:). Here's the [proposal draft](https://gist.github.com/moiseev/2f36376c8ef4c2b1273cff0bfd9c3b95).
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: "standart" -> "standard"

File: _drafts/2017-10-26-issue-92.md [hunk 5]
```diff
@@ -9,50 +9,92 @@ sponsor:
...
 
-> TODO
+[Rintaro Ishizaki](https://github.com/rintaro) [opened](https://github.com/apple/swift/pull/12457) merge request to fix [SR-5869: trailing closure not parsed correctly in if-statement conditional](https://bugs.swift.org/browse/SR-5869). The deal is: this fix allows users to write conditional statements with functions with trailing closures inside without parentheses. Before **if nums.filter ({ $0 % 2 == 0 }).isEmpty {**; after **if nums.filter { $0 % 2 == 0 }.isEmpty {**.
 
-### Rejected proposals
+[Doug Gregor](https://github.com/DougGregor) [fixed](https://github.com/apple/swift/pull/12633) compiler crash [SR-6100: Compilation hangs when catching error with associated value defined inside generic class/struct](https://bugs.swift.org/browse/SR-6100).
 
-> TODO
+[Mark Lacey](https://github.com/rudkx) [opened](https://github.com/apple/swift/pull/12631) merge request with implementation of warning in case of type is spelled as ImplicitlyUnwrappedOptional rather than T!. Seems [SE-0054: Abolish ImplicitlyUnwrappedOptional type](https://github.com/apple/swift-evolution/blob/master/proposals/0054-abolish-iuo.md) will get a full implementation soon!
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: any reason why we use **merge request** in some places and **pull request** in others? I think they should all be **pull request**.

Comment (RomanVolkov) [reply _drafts/2017-10-26-issue-92.md]: good point

File: _drafts/2017-10-26-issue-92.md [hunk 6]
```diff
@@ -9,50 +9,92 @@ sponsor:
     displaylink: TODO
 ---
 
-> TODO: intro comments
+Greetings! Hope you rested from Swift updates a little and ready to absorb Swift news again. Past two weeks please us with development activity at main swift repo. Plus, two exciting themes was covered by Swift Unwrapped. Enjoy the issue!
```
Comment (SD10) [inline _drafts/2017-10-26-issue-92.md]: "Greetings! Hope you're feeling a little rested from the week off and are ready to absorb Swift news again. There has been some interesting development activity at the main Swift repo over the past two weeks. Plus, two exciting themes was covered by Swift Unwrapped. Enjoy the issue!"

Let me know what you think of that 😅 I'm done now~

**Comment by jessesquires** [comment]
Thanks @SD10 ! 👍 💯

**Comment by BasThomas** [comment]
@jessesquires don't forget to move it to `_posts` :)

**Comment by jessesquires** [comment]
@BasThomas done! 🥇

**Pull request merged**